packages/avatars/demo/statusindicator.stories.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { StatusIndicator } from '@zendeskgarden/react-avatars';
+
+import { StatusIndicatorStory } from './stories/StatusIndicatorStory';
+
+<Meta title="Packages/Avatars/StatusIndicator" component={StatusIndicator} />
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story
+    name="StatusIndicator"
+    argTypes={{
+      children: { control: 'text' }
+    }}
+    args={{
+      'aria-label': 'Label',
+      children: 'Offline'
```

Comment:
[nit]

```suggestion
      children: 'Status'
```

...to avoid potential confusion that the text is intrinsically tied to the `type`.